asset/css/crontab.less

```diff
@@ -0,0 +1,107 @@
+// Cron tab expression widget
+
+.crontab {
+  .ordinal {
+    display: flex;
+    flex-wrap: wrap;
+
+    .radio-label {
+      flex: 1 auto;
+    }
+
+    select {
+      flex: 1 1 auto;
+
+      &:not(select:last-of-type) {
+        margin-right: 1em;
+      }
+    }
+  }
+
+  .radio-label {
+    width: 100%;
+    margin-bottom: .5em;
+    display: flex;
+    align-items: center; // To center the radio element on safari
+  }
+
+  .number-specifier > input[type="number"] {
+    width: 5em;
+    margin: 0 1em;
+  }
+
+  .monthly, .annually, .ordinal {
+    padding: .4em;
+    border: .1em solid @crontab-list-border-color;
+    .rounded-corners(.6em);
+
+    li {
+      margin-bottom: .08em;
+    }
+  }
+
+  .list-items {
+    margin: 0;
+    padding: 0;
+
+    &.disabled { // When the "On the" radio button is checked
+      pointer-events: none;
+
+      label {
+        color: inherit;
+        background-color: @base-gray-lighter;
+      }
+    }
+
+    li {
+      list-style: none;
+      display: inline-flex;
+      width: calc(100% / 7);
+
+      label {
+        width: 100%;
+        cursor: pointer;
+        text-align: center;
+        padding: .8em 0 .8em;
+        border-right: .1em solid @crontab-list-border-color;
+        background: @crontab-selectable-list-bg;
+        color: @crontab-default-text-color;
+      }
+
+      &:hover {
+        background-color: @crontab-list-hover-color;
```

Comment:
This has no effect for me, the color is near transparent. Also, please think of keyboard only users which use TAB to navigate. Add `:focus` here to also show them what they've selected.